coderd/jobreaper/detector.go

```diff
@@ -25,6 +24,10 @@ const (
 	// before it is considered hung.
 	HungJobDuration = 5 * time.Minute
 
+	// PendingJobDuration is the duration of time since the last update to a job
+	// before it is considered hung.
+	PendingJobDuration = 30 * time.Minute
```

Comment:
Both of these comments should be changed.

`... last update to a RUNNING job before it is considered hung`.

`... last update to a PENDING job before it is considered dead.`